examples/models/file/basic_plot.py

```diff
@@ -1,3 +1,12 @@
+'''A scatter plot of a smooth periodic oscillation. This example demonstrates red circle scatter markers with black outlines, using the low-level bokeh.models API.
```

Comment:
```suggestion
''' A scatter plot of a smooth periodic oscillation. This example demonstrates red
circle scatter markers with black outlines, using the low-level ``bokeh.models``
API.
```
Just as a general convention, we start multiline comments with a single space. We also try to keep line length to 80 chars for paragraphs of text (I'll add that info to the original issue).